### DISCUSSION TITLE
EncodeTFBS getposition

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#1405] [Questions / Documentation]
Not sure if this is a bug, or if I am missing something here.
I added "EncodeTFBS":
```projHeme2 <- addArchRAnnotations(ArchRProj = projHeme2, collection = "EncodeTFBS")```
but when I try getting the motif position, I get this error:
<img width="697" alt="Screen Shot 2022-04-27 at 8 08 37 PM" src="https://user-images.githubusercontent.com/19545198/165668789-5cd2cdd6-e56b-499a-a6b0-e647d930cda5.png">

**Comment by rcorces** [comment]
This is an obscure one. Had to dig into the code a bit.
The difference is between `addPeakAnnotations()` and `addArchRAnnotations()`. The former adds a record for the actual position, the later does not. So, as written, none of the results from `addArchRAnnotations()` are compatible with `getPositions()`

https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/AnnotationPeaks.R#L759

vs

https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/AnnotationPeaks.R#L224

I'm not 100% sure what the original rationale for this would have been. Seems like something that should be addressed. I'm moving this to Issues.
